lib/atm.rb

```diff
@@ -11,11 +11,12 @@ def initialize
 
    def withdraw(amount, pin, expiry, account)
       case
-      when negative_value(amount) then return_error_message(:negative_amount)
-      when !check_pin(pin, account.pin) then return_error_message(:wrong_pin)
+      when negative_value?(amount) then return_error_message(:negative_amount)
+      when incorrect_pin?(pin, account.pin) then return_error_message(:wrong_pin)
       when check_expiry(expiry, account.exp_date) then return_error_message(:card_expired)
-      when !sufficient_fund_in_atm(amount) then return_error_message (:no_sufficient_fund)
-      when amount % 5 != 0 then return_error_message (:non_rounded_amount)
+      when no_sufficient_fund_in_atm?(amount) then return_error_message :no_sufficient_fund
```

Comment:
Metrics/LineLength: Line is too long. [91/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)